layer/profiles.cpp

```diff
@@ -3688,6 +3917,153 @@ bool JsonLoader::GetFormat(const Json::Value &formats, const std::string &format
     return valid;
 }
 
+bool QueueFamilyMatch(const VkQueueFamilyProperties &device_qfp, const VkQueueFamilyProperties &profile_qfp) {
+    if ((device_qfp.queueFlags & profile_qfp.queueFlags) != profile_qfp.queueFlags) {
+        return false;
+    }
+    if (device_qfp.queueCount < profile_qfp.queueCount) {
+        return false;
+    }
+    if (device_qfp.timestampValidBits != profile_qfp.timestampValidBits) {
```

Comment:
This should be `<`, not `!=`. While the `vk.xml` is missing the corresponding limit type this is actually a minimum maximum.